internal/common/metrics.go

```diff
@@ -105,11 +119,18 @@ func sendStatsdMetrics(metricPerMinute int, duration time.Duration) error {
 	defer ticker.Stop()
 	endTimeout := time.After(duration)
 
+	// Sending the statsd metric within the first minute before the ticker kicks in the next minute
```

Comment:
Whenever we call `sendStatsdMetrics`, we create a ticker for every [60 second](https://github.com/aws/amazon-cloudwatch-agent-test/pull/165/files#diff-781bb3810ede78771b4deb63bb6bb797a2d06434f0d844429b5ad16a20304e0fR118), however; the first call (e.g from 0 to 1 minutes) won't be sent; therefore, start sending statsd metrics here during the first time for and after every 60s, we will send statsd metrics regualarly
```
case <-ticker.C:
			for t := 0; t < metricPerMinute; t++ {
				client.Inc(fmt.Sprint(t), int64(t), 1.0)
				client.Count(fmt.Sprint(t), int64(t), []string{}, 1.0)
			}
```